### DISCUSSION TITLE
Scons ignoring --prefix

### DISCUSSION THREAD

**Discussion by DatanoiseTV** [roc-streaming/roc-toolkit#443] [Q&A]
I'm building for a Buildroot-based Linux distribution and I've made a package.
Unfortunately, the Build is done always in build and bin and --prefix=$(TARGET_DIR) gets ignored.

**Comment by gavv** [comment]
Hi, thanks for report.

`--prefix` options defines where to *install* files. You usually run:

```
scons <SOME_OPTIONS>
sudo scons <SAME_OPTIONS> --prefix=<DIR> install
```
